wordpress-mobile/WordPress-Android
# Added fullstops to Subtitle String for the Quick start

Fixes #

Added fullstops to Subtitle String for the Quick start Tasks

To test:

1. Create a new account.
2. Check the quick start tasks.

## Regression Notes
1. Potential unintended areas of impact


3. What I did to test those areas of impact (or what existing automated tests I relied on)


4. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
